test/support/acceptance_case.ex

```diff
@@ -0,0 +1,79 @@
+defmodule PRM.Support.AcceptanceCase do
+  @moduledoc """
+  Acceptance test helper
+  """
+  use ExUnit.CaseTemplate
+
+  using do
+    repo =
+      case System.get_env("CONTAINER_HTTP_PORT") do
+        nil -> PRM.Repo
+        _   -> nil
+      end
+
+    quote do
+      use EView.AcceptanceCase,
+        async: false,
+        otp_app: :prm,
+        endpoint: PRM.Web.Endpoint,
+        repo: unquote(repo),
+        headers: [{"content-type", "application/json"}]
+
+      def assert_error(%HTTPoison.Response{body: %{} = body} = response, code, entry, type) do
+        assert %{
+          "meta" => %{"code" => ^code},
+          "error" => %{
+            "message" => _,
+            "type" => "validation_failed",
+            "invalid" => [%{
+              "entry" => ^entry,
+              "entry_type" => ^type,
+              "rules" => ["required"]
+            }]
+          },
+        } = body
+
+        response
+       end
+
+      def assert_404(%HTTPoison.Response{body: %{} = body} = response) do
+        assert %{
+          "meta" => %{
+            "code" => 404,
+            "request_id" => _,
+            "type" => "object",
+            "url" => _
+          },
+          "error" => %{
+            "type" => "not_found"
+          }
+        } = body
+
+        response
+       end
+
+      def assert_422(%HTTPoison.Response{body: %{} = body} = response) do
```

Comment:
Don't match map here, assertion below would do all the work.

Why do we need this special-cases methods, when assert status is sufficient. You don't only need to assert that response "looks like validation response", but also a list of invalid fields with validation response.